src/riak_core_app.erl

```diff
@@ -145,3 +146,12 @@ register_capabilities() ->
       end,
       Capabilities),
     ok.
+
+%% TODO: This belongs in riak_kv - an issue will be created to move it, but time constraints.
```

Comment:
Is `eleveldb` is always started and available before `riak_core` started? I'm thinking of when we don't use a different backend e.g., `bitcask`?